docs/docs/guides/upgrade-to-v5.md

```diff
@@ -60,7 +61,7 @@ import { auth } from "../path-to-config/auth"
 
 The old configuration file, contained in the API Route (`pages/api/auth/[...nextauth].ts`), now becomes a 1-line handler for `GET` and `POST` requests for those paths.
 
-```ts title="app/api/auth/[...nextauth]/route.ts"
+```ts title="app/api/auth/[[...nextauth]]/route.ts"
```

Comment:
Where did you get this change from? Since brackets still work with Auth.js v5 and Next.js 14/app router, see: https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes